editor/editor_node.cpp

```diff
@@ -3574,6 +3574,14 @@ int EditorNode::new_scene() {
 }
 
 Error EditorNode::load_scene(const String &p_scene, bool p_ignore_broken_deps, bool p_set_inherited, bool p_clear_errors, bool p_force_open_imported, bool p_silent_change_tab) {
+	for (Ref<TextFile> file : ScriptEditor::get_singleton()->get_open_text_files()) {
```

Comment:
Need to handle shadowing here

Reply:
Changed it to opened_text_file, hopefully that fixes it.